### ISSUE TITLE
Problems with momentum conservation in the prototype

### ISSUE THREAD

**Issue by ariel-phet** [phetsims/my-solar-system#159]
The same users who has thoroughly tested the prototype (and found the error in the value of G) reported the following:

> I finally got a chance to completely check out the new My Solar System. It worked very well for the activity I wrote for the old one. I am updating it now, hopefully in time for some teachers to use it. There were 2 things I wanted you to know. When objects collided in the old version they combined and momentum was conserved. I used this for the last activity. When I try it with the new one something really odd happens. The less massive object completely disappears but transfers its momentum to the other one. This is physically impossible. I am hoping since this is still a prototype that you can fix this.

Seems like this issue should be addressed as it means there is incorrect physics being presented

**Comment by AgustinVallejo** [comment]
I remember we had decided not to merge the two masses as you could technically keep adding bodies and crashing them, creating a body that was beyond the allowed mass scale of the sim... However, I see how it can be problematic in a physical sense. We could implement the merge and have it so you can't add more bodies after exceeding the maximum mass.

**Comment by AgustinVallejo** [comment]
Discussing more with @arouinfar and @DianaTavares, there's some observations:
- The old sim merges the masses but doesn't make the correct momentum or energy conservation calculations.
- Allowing mass merging would put the possible body mass way above the maximum slider value. Even if we limit the number of bodies as said in my previous comment, it could go up to 1200 units of mass, where the current maximum is 300.
- There's some PhET-IO implications of limiting the mass anyways.

We see two ways forward:
1. Leave the sim as it is now and let them know that it's not supposed to be used as a reference for collisions, only gravitational dynamics. Right now the sim respects the conservation of momentum, and the missing mass could be explained as "debris lost in outer space". Energy wouldn't be conserved in this case but it's not unusual.
2. Allow for mass merging by implementing a model of `inputMassProperty `and `actualMassProperty `(or `massProperty `and `mergedMassProperty`) this way having a distinction between what the user can input, and what the sim can produce (Density also does something similar). This would help with the future PhET-IO implications.

We like (2) better and will discuss further with other devs to see if this can easily be implemented.

**Comment by DianaTavares** [comment]
Just for the record, We played in the meeting with Flash to observe it behavior when the bodies crash. The simulation just adds the masses and puts the velocity to zero. 

Objects with different masses. 
![different_mass](https://user-images.githubusercontent.com/26031501/232578549-eb853fb6-8cf2-4b09-b32f-96f4c7ec6dbb.gif)

Two objects same masses:
![sameMass](https://user-images.githubusercontent.com/26031501/232585277-d78c8a6e-3df9-4757-9da3-dee96a355521.gif)

Three objects:
![moregif](https://user-images.githubusercontent.com/26031501/232584983-1f14d2fb-4490-4177-8be0-73d64da12dc0.gif)

I found a system where the final velocity of a crash is not zero, but when I play it to a slower, it ends with a velocity of zero: 
![noZero](https://user-images.githubusercontent.com/26031501/232586580-9e595a3b-9ef4-4d48-a990-a6b40f9fc6b6.gif)

**Comment by arouinfar** [comment]
@ariel-phet can you review and convey this information to the user? If our conclusion is unacceptable or you have further thoughts, we are also open to further discussion.

**Comment by AgustinVallejo** [comment]
Todays design meeting proposed the following implementation:
> Don't add up masses, and instead of adding the momentum of the smaller body to the bigger, add half of it, to suggest some momentum was transferred to debris.

However, when trying to implement it, I got into some unusual physics behavior. To summarize:
- First, and this is also happening before the change: Momentum from the small body gets transferred... However, if the small body has v=0 and the bigger one slams into it, there is no momentum to give, so the bigger body keeps going without breaking. In the current state, it is "correct" because we put priority into momentum conservation and that situation covers it... However, if we want to manifest that some momentum is lost as debris, in the mentioned case 0 momentum would be lost.
- Second, when two bodies start at v=0, they attract eachother and collide. In the current state, as well as in the real world, the resultant momentum of the system is still 0 regardless of the masses. Because F=dP/dt, and the forces are equal, both bodies will have always equal an opposing momentums. BUT if we implement the change of adding half of the momentum, that equality is broken and there is actually momentum being added to the system after the collision!! Making the sim even less realistic.
- In fact, because of the above point, it becomes almost impossible to make any prediction to momentum transfer, unless manually collided, because gravity starts adding momentum in any collision.

I think it's more cautious to leave the collisions as they are right now, and add the disclaimer that collisions won't be realistic as we decided not to merge masses.
